src/samples/azureactivedirectoryauthentication/src/main/java/AzureActiveDirectoryAuthentication.java

```diff
@@ -46,7 +46,7 @@ public static void main(String[] args) {
 
             System.out.println("For testing 'ActiveDirectoryIntegrated' Authentication, do one of the following:");
             System.out.println("   1. Generate Kerberos Ticket and validate its availability with klist tool, or");
-            System.out.println("   2. Place sqljdbc_auth.dll in the same directory as the pom.xml file.");
+            System.out.println("   2. Place mssql-jdbc_auth in the same directory as the pom.xml file.");
```

Comment:
Same as my previous comment. We should probably spell out the full mssql-jdbc_auth filename to make it obvious to users what we have done.